Jellyfin.Plugin.KodiSyncQueue/Data/DbRepo.cs

```diff
@@ -6,411 +6,201 @@
 using MediaBrowser.Model.Serialization;
 using Jellyfin.Plugin.KodiSyncQueue.Entities;
 using System.Text;
+using LiteDB;
 using MediaBrowser.Model.IO;
 using Microsoft.Extensions.Logging;
 
 namespace Jellyfin.Plugin.KodiSyncQueue.Data
 {
     public class DbRepo: IDisposable
     {
-        private readonly object _createLock = new object();
-        private readonly object _userLock = new object();
-        private readonly object _folderLock = new object();
-        private readonly object _itemLock = new object();
+        private readonly LiteDatabase _liteDb;
+        private const string ItemsCollection = "items";
+        private const string UserInfoCollection = "user_info";
 
-        private const string dbFolder = "Jellyfin.Plugin.KodiSyncQueue.F.1.40.json";
-        private const string dbItem = "Jellyfin.Plugin.KodiSyncQueue.I.1.40.json";
-        private const string dbUser = "Jellyfin.Plugin.KodiSyncQueue.U.1.40.json";
-                
-        private string dataPath = "";        
-        
-        public string DataPath
-        {
-            get { return dataPath; }
-            set { dataPath = Path.Combine(value, "SyncData"); }
-        }
-
-        private NanoApi.JsonFile<FolderRec> folderRecs = null;
-        private NanoApi.JsonFile<ItemRec> itemRecs = null;
-        private NanoApi.JsonFile<UserInfoRec> userInfoRecs = null;
-
-        private static DbRepo instance = null;
-        public static ILogger logger = null;
-        public static IJsonSerializer json = null;
-        public static string dbPath = "";
-        public static IFileSystem fileSystem = null;
-
-        public static DbRepo Instance
-        {
-            get
-            {
-                if (instance == null)
-                {
-                    instance = new DbRepo(dbPath);
-                }
-                return instance;
-            }
-        }
+        private readonly ILogger _logger;
+        private readonly IJsonSerializer _json;
 
+        public static DbRepo Instance;
```

Comment:
shouldn't this be a public getter/private setter thing? or even better a proper singleton pattern?